content/eaas/samples/python_sample.py

```diff
@@ -0,0 +1,8 @@
+import requests
+
+headers = {'Authorization': 'Bearer PLACE_JWT_HERE'}
+req_response = requests.get('https://api-eus.qrypt.com/api/v1/quantum-entropy',

```

Comment:
There should be a parameter also passed with this request:
params = {"size": INT}   # INT is a value of range 1 - 512